omod/src/main/java/org/openmrs/module/fhir2/web/filter/AuthenticationFilter.java

```diff
@@ -58,10 +58,8 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
 							return;
 						}
 					} else {
```

Comment:
@Ruhanga what happens for this else after removing the bellow lines

Reply:
@HerbertYiga, the request is allowed through to other filters.